pyproject.toml

```diff
@@ -45,9 +46,10 @@ ansys-tools-filetransfer = ">=0.1"
 networkx = ">=3.0.0"
 
 # Optional dependencies which are included in the 'extras' section.
-# Update also the 'dev' group when these are updated.
-pyvista = { version = ">=0.42.0", optional = true, extras = ["jupyter", "trame"] }
-ansys-mapdl-core = { version = ">=0.68.3", optional = true }
+# TODO: Once ansys-mapdl-core supports Python 3.13
+# remove the 'python' limit below and update the main Python version in CI.
+pyvista = { version = ">=0.42.0,!=0.42.2", optional = true, extras = ["jupyter"] }
+ansys-mapdl-core = { version = ">=0.68.3", optional = true, python=">=3.10,<3.13" }
```

Comment:
You should be unblocked now @greschd! 🥳

Reply:
Party time! 🎉

Reply:
<img src="https://media1.giphy.com/media/v1.Y2lkPWJkM2VhNTdlZW9wNTNxcmlncGdxamI0cWVpM2s3NG83N2dpbmhybHlscWxnZWdwdyZlcD12MV9naWZzX3NlYXJjaCZjdD1n/IwAZ6dvvvaTtdI8SD5/giphy.gif"/>